packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -243,6 +294,9 @@ export default class RunPanelStore {
             // Bot should indicate it stopped in below cases:
             // - When error happens and it's an unrecoverable error
             indicateBotStopped();
+            this.run_limit = '';
+            this.max_losses = '';
+            this.set_max_losses = '';
```

Comment:
Can we use the function you created to reset these values? i.e. `initialSelfExclustion`? (please update this function name)

Reply:
> Can we use the function you created to reset these values? i.e. `initialSelfExclustion`? (please update this function name)

Done